src/main/java/socialite/ui/PersonCard.java

```diff
@@ -70,19 +73,104 @@ public PersonCard(Person person, int displayedIndex) {
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
         remark.setText(person.getRemark().value);
-        tiktokIcon.setImage(new Image(this.getClass().getResourceAsStream("/images/tik-tok.png")));
-        tiktok.setText(" @" + person.getTiktok().value + " ");
-        twitterIcon.setImage(new Image(this.getClass().getResourceAsStream("/images/twitter.png")));
-        twitter.setText(" @" + person.getTwitter().value + " ");
+        this.makeHandle(person.getFacebook(), Platform.FACEBOOK);
+        this.makeHandle(person.getInstagram(), Platform.INSTAGRAM);
+        this.makeHandle(person.getTelegram(), Platform.TELEGRAM);
+        this.makeHandle(person.getTiktok(), Platform.TIKTOK);
+        this.makeHandle(person.getTwitter(), Platform.TWITTER);
+
         person.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
-        facebookIcon.setImage(new Image(this.getClass().getResourceAsStream("/images/facebook.png")));
-        facebook.setText(" @" + person.getFacebook().value + " ");
-        instagramIcon.setImage(new Image(this.getClass().getResourceAsStream("/images/instagram.png")));
-        instagram.setText(" @" + person.getInstagram().value + " ");
-        telegramIcon.setImage(new Image(this.getClass().getResourceAsStream("/images/telegram.png")));
-        telegram.setText(" @" + person.getTelegram().value + " ");
+    }
+
+    private void openBrowser(String url) {
```

Comment:
Maybe we can leave a reference to the place where this code was from, since not everyone would know what each line means.